content/country/spain/index.en.md

```diff
@@ -7,15 +7,16 @@ country: "spain"
 
 ## FIP Information
 
-In Spain, using FIP is not always straightforward, as simply boarding and riding along rarely works. On most routes, a ticket at the FIP global fare must be booked in advance. Since only as many tickets are sold as there are available seats, this purchase should not be made at the last minute (even for regional trains). [Renfe]({{< ref "/operator/renfe" >}} "Renfe") is the largest railway operator in Spain and accepts FIP. On lucrative routes, it competes with many private providers like OUIGO or Iryo, which do not accept FIP. Another provider in the Basque Country is Euskotren, which also accepts FIP.
+In Spain, using FIP is not always straightforward, as simply boarding and riding along rarely works. On most routes, a ticket at the FIP Global Fare must be booked in advance. Since only as many tickets are sold as there are available seats, this purchase should not be made at the last minute (even for regional trains). [Renfe]({{< ref "/operator/renfe" >}} "Renfe") is the largest railway operator in Spain and accepts FIP. On lucrative routes, it competes with many private providers like OUIGO or Iryo, which do not accept FIP. Another provider in the Basque Country is Euskotren, which also accepts FIP.
 
 ## Insteresting
 
 Spain does not have a particularly dense rail network; instead, it mainly consists of new high-speed standard gauge lines and older regional lines in Iberian broad gauge. The focus is on (mostly fast) connections between major cities. In addition, there are suburban trains in and around the metropolitan areas that ensure dense local transport.
 
-Stations usually have ticket barriers, which require a valid digital or analog ticket on a card to pass through. With FIP free travel coupons, staff on site must be approached so they can manually open the barriers. Furthermore, for long-distance travel, luggage checks are conducted at the stations. While not as strict as at airports, they still take time. Therefore, it is recommended to arrive at the station at least 30 minutes before departure for such trips.
+Stations usually have ticket barriers, which require a valid digital or analog ticket on a card to pass through. With FIP Coupons, staff on site must be approached so they can manually open the barriers. Furthermore, for long-distance travel, luggage checks are conducted at the stations. While not as strict as at airports, they still take time. Therefore, it is recommended to arrive at the station at least 30 minutes before departure for such trips.
 
 ## Operators without FIP
 
 - Iryo
 - OUIGO
+- Avlo
```

Comment:
Avlo is not an operator, it's a train category that is operated by renfe. I'd not list it here.

Reply:
Yeah I know and I was really thinking about this. However I think in the perspective of many customers it could be seen as an own """company""". It's like OuiGo, which is technically SNCF. I would write it down just to be sure. Would that be okay for you?

Reply:
I'm fine with it, but we should probably consider changing the title to "FIP facilities are not valid on" (how Rail Delivery Guide does it) or similar. But this should be consistent across all pages, so let's consider it for another PR.